packages/hdwallet-metamask-shapeshift-multichain/src/adapter.ts

```diff
@@ -1,14 +1,13 @@
-import detectEthereumProvider from "@metamask/detect-provider";
```

Comment:
We obviously don't need this anymore since we support MIPD

Reply:
Actually we do

Reply:
?